speechbrain/lobes/models/convolution.py

```diff
@@ -264,3 +283,71 @@ def forward(self, x):
             out = out + self.reduce_conv(x)
             out = self.drop(out)
         return out
+
+    def get_filter_properties(self) -> FilterProperties:
+        return stack_filter_properties(self.filter_properties)
+
+
+class ConformerFeatureExtractorWrapper(torch.nn.Module):
```

Comment:
I don't like having this class. This may, in the future, imply a shit ton of complexity as it will grow with the number of supported model. Why do we need this? I don't see anything special about the forward.

Reply:
I still understand the point of it. Could you elaborate?

Reply:
The point is that I don't see how to abstract this only in the YAML. The alternative would be to make `StreamingASR` aware of how to call the Conformer feature extractor which would unnecessarily be model-specific.

These individual submodules are specific to the Conformer so forcing other models to use those would seem wrong.  
I don't think it can simply be defined as a module list in the YAML because of how `lens`, etc. need to be passed around for only some of these modules...

Reply:
If I am not mistaking we have a length capable sequential just for that, that can be used in the yaml. @pplantinga might know more. This wrapper really looks unnecessary :-(

Reply:
> If I am not mistaking we have a length capable sequential just for that, that can be used in the yaml. @pplantinga might know more. This wrapper really looks unnecessary :-(

Actually, yeah, this works. I had ruled it out for a reason I forgot was fixed (the default `epoch=None` param). LengthsCapableSequential seems to work, fixing.